README.md

```diff
@@ -110,17 +110,17 @@ We're mostly a Rails app, with a bit of Webpack sprinkled in. **For most cases,
 
 Here are some singleton commands you may need, usually in a separate instance/tab of your shell.
 
-- Running the job server (if using `bin/rails server`) -- this is mostly for notifications and emails: **`bin/rails jobs:work`**
-- Clearing jobs (in case you don't want to wait for the backlog of jobs): **`bin/rails jobs:clear`**
+- Running the job server (if using `bin/rails server`) -- this is mostly for notifications and emails: **`bin/rails jobs: work`**
+- Clearing jobs (in case you don't want to wait for the backlog of jobs): **`bin/rails jobs: clear`**
 
-Current gotchas: potential environment issues with external services need to be worked out.
+Current gotchas: potential environmental issues with external services need to be worked out.
 
 #### Suggested Workflow
 
 We use [Spring](https://github.com/rails/spring), and it is already included in the project.
 
-1.  Use the provided bin stubs to start Spring automatically, i.e. `bin/rails server`, `bin/rspec spec/models/`, `bin/rails db:migrate`.
-1.  If Spring isn't picking up on new changes, use `spring stop`. For example, Spring should always be restarted if there's a change in environment key.
+1.  Use the provided bin stubs to start Spring automatically, i.e. `bin/rails server`, `bin/rspec spec/models/`, `bin/rails db: migrate`.
```

Comment:
`db: migrate` should not have a space in it, again Grammarly will recommend this, but it doesn't know anything about backticks indicating code.